cost/azure/old_snapshots/azure_delete_old_snapshots.pt

```diff
@@ -515,6 +517,9 @@ EOS
         label "ID"
         path "resourceID"
       end
+      field "size" do
+        label "Size"
```

Comment:
I would add (GB) to the label to make  it clear how size is being measured:

field "size" do
        label "Size (GB)"